src/ucp/rma/rma_send.c

```diff
@@ -252,8 +258,10 @@ ucs_status_t ucp_get_nbi(ucp_ep_h ep, void *buffer, size_t length,
     ucp_ep_rma_config_t *rma_config;
     ucs_status_t status;
 
-    UCP_RMA_CHECK_PARAMS(ep->worker->context, buffer, length, status,
-                         return status);
+    UCP_CONTEXT_CHECK_FEATURE_FLAGS(ep->worker->context, UCP_FEATURE_RMA,
```

Comment:
let's define UCP_RMA_CHECK(..) and UCP_RMA_CHECK_PTR(...) to reduce some code duplication